cpp-ch/local-engine/Functions/SparkFunctionDecimalBinaryOperator.h

```diff
@@ -23,8 +23,16 @@
 
 using BitInt128 = signed _BitInt(128);
 using BitUInt128 = unsigned _BitInt(128);
+#if defined(__x86_64__)
 using BitInt256 = signed _BitInt(256);
 using BitUInt256 = unsigned _BitInt(256);
+#else
+// up to version 18, clang supports large _Bitint sizes on x86 and x86-64; 
```

Comment:
@loudongfeng if we switch to clang 19, does it supports 256 bits?

Reply:
Latest code still has this limitation.  https://github.com/llvm/llvm-project/blob/main/clang/test/SemaCXX/ext-int.cpp#L32